dvc_azure/__init__.py

```diff
@@ -116,7 +116,9 @@ def _prepare_credentials(self, **config):
             )
 
         any_secondary = any(
-            value for key, value in login_info.items() if key != "account_name"
+            value
+            for key, value in login_info.items()
+            if key not in ("account_name", "version_aware")
```

Comment:
The code in `_prepare_credentials` is trying to be too smart, and this particular trick I also don't really like and would've prefered an explicit secondary auth options list, but this will do the trick for now I guess.